openfisca_aotearoa/variables/demographics/relationships.py

```diff
@@ -11,7 +11,7 @@ class has_a_partner(Variable):
     value_type = bool
     entity = Person
     label = u"Is this person in a relationship?"
-    definition_period = MONTH  # This variable cannot change over time.
+    definition_period = MONTH  # This variable changes over time.
```

Comment:
I don't understand why definition period changes ...

Reply:
the definition period doesn't change.

has_a_partner is a Variable. Whether you have a partner or not will change over time.